packages/create-plugin/src/constants.ts

```diff
@@ -41,9 +41,7 @@ export enum PLUGIN_TYPES {
 
 // This gets merged into variables coming from user prompts (when scaffolding) or any other dynamic variables,
 // and will be available to use in the templates.
-// Example: "@grafana/ui": "{{ grafanaVersion }}"
 export const EXTRA_TEMPLATE_VARIABLES = {
-  grafanaVersion: '10.3.3',
```

Comment:
I guess this one is causing the build to fail. So wee need to fill that with a value or fetch this value on a different place where we consume this variable.